backend/backend/handlers/metadata/create.py

```diff
@@ -0,0 +1,24 @@
+import json
+import traceback
+
+from backend.handlers.metadata import logger, mask_sensitive_data, build_response, create_or_update, validate_event, validate_body, ValidationError
+
+
+def lambda_handler(event, context):
+    logger.info(mask_sensitive_data(event))
+    try:
+
+        validate_event(event)
+        
+        body = validate_body(event)
+        assetId = event['pathParameters']['assetId']
+
+        create_or_update(assetId, body['metadata'])
+
+        return build_response(200, json.dumps({ "status": "OK" }))
+    except ValidationError as ex:
+        logger.info(traceback.format_exc())
```

Comment:
logger.error?

Reply:
It's a client input validation error opposed to an application level error. Might be good for us to define how we use different log levels. I've often used conventions like these: 

- debug: chatty details that are helpful to understanding during development
- info: any messages relating to nominal operation of the application, like a business transaction occurred, client validation failure
- warn: something that an operator may want to know about for tuning purposes... like, free memory or disk volume usage falling below threshold for safe operation. 
- error: business transaction failure due to server-side faults, if too many of these happen, alert operators
- fatal: this application is about to crash, wake somebody up if necessary :) 

Maybe the `ValidationError` class could have a different name to increase some clarity.